.github/workflows/test-notify-slack.yml

```diff
@@ -0,0 +1,48 @@
+name: 'Run notify-slack action'
+
+on:
+  pull_request:
+    paths:
+      - actions/notify-slack/*
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  test-notify-slack-action:
+    name: 'Uses the notify-slack action'
+    runs-on: ubuntu-latest
+    if: always()
+    outputs:
+      status: ${{ steps.notify.outputs.status }}
```

Comment:
What is this output for?

Reply:
To output the workflow status

Reply:
I do not understand.  We are not using this output anywhere.  There are no jobs that follow this so it cannot be used.  This is a _job_ level output being declared here.

Reply:
I thought we could use the output to get the status of the workflow like we did for the engagement